webrender/src/renderer.rs

```diff
@@ -1772,10 +1772,8 @@ impl<B: hal::Backend> Renderer<B> {
             match msg {
                 #[cfg(not(feature = "gleam"))]
                 ResultMsg::UpdateWindowSize(window_size) => {
```

Comment:
If we don't need `UpdateWindowSize`, we can delete it here as well, also from `ResultMsg`.